tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/upgrade/UpgradeAssistantSettingsUtil.java

```diff
@@ -85,6 +85,22 @@ public boolean accept( File dir, String name )
         return null;
     }
 
+    public static <T> T getObjectFromStore( Class<T> clazz, String suffix ) throws IOException
```

Comment:
for 1,as we didn't do any update actions for the json files and we just "get" at before.